client/src/widgets/StyledTextField.js

```diff
@@ -3,12 +3,14 @@ import TextField from 'material-ui/TextField'
 import { style } from './styles'
 
 export const StyledTextField = props => {
-  const { text, multiLine, disabled } = props
+  const { text, multiLine, disabled, org } = props
   const isMultiLine = !!multiLine
```

Comment:
isn't this the same as writing just `multiLine`?

Reply:
yes....I don't remember why I thought it was needed at the time, but I'm just going to remove that and use the passed in multiLine prop